eZ/Publish/Core/FieldType/Image/ImageThumbnailStrategy.php

```diff
@@ -43,12 +43,16 @@ public function getFieldTypeIdentifier(): string
 
     public function getThumbnail(Field $field, ?APIVersionInfo $versionInfo = null): ?Thumbnail
     {
-        /** @var \eZ\Publish\SPI\Variation\Values\ImageVariation $variation */
-        $variation = $this->variationHandler->getVariation(
-            $field,
-            $versionInfo ?? new VersionInfo(),
-            $this->variationName
-        );
+        try {
+            /** @var \eZ\Publish\SPI\Variation\Values\ImageVariation $variation */
+            $variation = $this->variationHandler->getVariation(
+                $field,
+                $versionInfo ?? new VersionInfo(),
+                $this->variationName
+            );
+        } catch (\Exception $e) {
+            return new Thumbnail();
```

Comment:
According to Thumbnail contract, resource is guaranteed to be a string:

```
class Thumbnail extends ValueObject
{
    /**
     * Can be target URL or Base64 data (or anything else).
     *
     * @var string
     */
    protected $resource;
    
    # ...
}
    
```

Reply:
Is there also any reason why we cant return null in that case as it should be handled according to typehint?

Reply:
Well, we could use `null` as I have done originally but it crashes in the other bundles, for example here: https://github.com/ezsystems/ezplatform-rest/blob/1.3/src/lib/Server/Output/ValueObjectVisitor/Version.php#L140

as the `Thumbnail` is proxy object at this point, not sure about other places.

Reply:
Dont get it, we return null but we somehow ends with Proxy object? If yes I would say that also needs a fix. And if our bundles are not ready to handle `null`s when this is in contract that also needs a fix.

Reply:
That's exactly the case, I'll change it to `null` in this case and handle remaining bundles.